odoo/tests/common.py

```diff
@@ -777,7 +778,8 @@ def _complete_traceback(self, initial_tb):
         # method since the error does not comme especially from the test method.
         while tb:
             code = tb.tb_frame.f_code
-            if code.co_filename.endswith('/unittest/case.py') and code.co_name in ('_callTestMethod', '_callSetUp', '_callTearDown', '_callCleanup'):
+            filename = code.co_filename.replace('\\', '/')
+            if filename.endswith('/unittest/case.py') and code.co_name in ('_callTestMethod', '_callSetUp', '_callTearDown', '_callCleanup'):
```

Comment:
In master this part of the code moved in a case.py file and actually checks that  `filename.endswith('/case.py')`

In the same spirit has the other suggestion, we could use `Path(code.co_filename).name == 'case.py'`

Reply:
Sounds good to me. I'll make the change.